network/simulation/node.go

```diff
@@ -148,6 +153,29 @@ func (s *Simulation) AddNodes(count int, opts ...AddNodeOption) (ids []enode.ID,
 	return ids, nil
 }
 
+// AddBootNode creates a bootnode using AddNode(opts) and appends it to Simulation.bootNodes
+func (s *Simulation) AddBootNode(opts ...AddNodeOption) (id enode.ID, err error) {
+	opts = append(opts, AddNodeAsBootNode(true))
+	id, err = s.AddNode(opts...)
+	if err != nil {
+		return id, err
+	}
+
+	return id, nil
+}
+
+// AddBootNodes creates count number of bootnodes using AddNodes(count, opts)
+// and appends them to Simulation.bootNodes
+func (s *Simulation) AddBootNodes(count int, opts ...AddNodeOption) (ids []enode.ID, err error) {
```

Comment:
for which reason do we need to add multiple bootnodes in one go? also, if we really need this functionality, why doesn't it reuse `AddBootnode()`

Reply:
To be honest, there probably isn't a huge reason. I mainly added it for the sake of following the pattern of other helper functions.

I could maybe see a scenario where network discovery is tested by having clusters connect to different bootnodes and checking health/time as the clusters discover each other?

Not sure if that's really a valid thing to test, so i'm happy to omit this function.

Reply:
I ended up removing this method.